app/src/main/java/org/piramalswasthya/cho/model/PatientVisitInfoSync.kt

```
@@ -99,7 +102,21 @@
         pharmacistDataSynced = SyncState.SYNCED,
-        visitCategory = benFlow.VisitCategory ?: ""
+        visitCategory = benFlow.VisitCategory ?: "",
+        visitDate = benFlow.visitDate?.let {
+            try {
+                val inputFormat = SimpleDateFormat("MMM d, yyyy, h:mm:ss a", Locale.ENGLISH)
+                val outputFormat = SimpleDateFormat("yyyy-MM-dd", Locale.getDefault())
+
+                val parsedDate = inputFormat.parse(it)
+                val onlyDateString = parsedDate?.let { d -> outputFormat.format(d) }
+                onlyDateString?.let { s -> outputFormat.parse(s) }
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)